src/components/VirtualKeyboard.tsx

```diff
@@ -47,27 +47,79 @@ const Keyboard = styled.div`
 interface Props {
   onKeyPress: PointerEventHandler
   keyDisabled?(key: string): boolean
+  keyMap?: KeyMap
 }
 
-const VirtualKeyboard = ({ onKeyPress, keyDisabled }: Props) => (
+interface Key {
+  label: string
+  ariaLabel?: string
+}
+
+interface KeyMap {
+  rows: Key[][]
+}
+
+const US_ENGLISH_KEYMAP: KeyMap = {
+  rows: [
+    [
+      { label: 'Q' },
+      { label: 'W' },
+      { label: 'E' },
+      { label: 'R' },
+      { label: 'T' },
+      { label: 'Y' },
+      { label: 'U' },
+      { label: 'I' },
+      { label: 'O' },
+      { label: 'P' },
+    ],
+    [
+      { label: 'A' },
+      { label: 'S' },
+      { label: 'D' },
+      { label: 'F' },
+      { label: 'G' },
+      { label: 'H' },
+      { label: 'J' },
+      { label: 'K' },
+      { label: 'L' },
+      { label: "'", ariaLabel: 'single-quote' },
+      { label: '"', ariaLabel: 'double-quote' },
+    ],
+    [
+      { label: 'Z' },
+      { label: 'X' },
+      { label: 'C' },
+      { label: 'V' },
+      { label: 'B' },
+      { label: 'N' },
+      { label: 'M' },
+      { label: ',', ariaLabel: 'comma' },
+      { label: '.', ariaLabel: 'period' },
+      { label: '-', ariaLabel: 'dash' },
+    ],
+    [{ label: 'space' }, { label: '⌫ delete' }],
```

Comment:
`⌫ delete` reads as "erase to the left, delete", which is a bit verbose but is okay IMO.

Reply:
Thanks for clarifying. Perhaps commit this a comment with this so that someone doesn't add and `ariaLabel` ?

Reply:
I decided to add an `ariaLabel` with just `delete` since that's how it reads it on Linux now. On macOS it reads as "erase to the left, delete".